cmake/OpenCVDetectCUDAUtils.cmake

```diff
@@ -109,7 +109,7 @@ macro(ocv_initialize_nvidia_device_generations)
   set(_arch_ampere   "8.0;8.6")
   set(_arch_lovelace "8.9")
   set(_arch_hopper   "9.0")
-  set(_arch_blackwell "10.0;12.0")
+  set(_arch_blackwell "10.0;10.3,11.0;12.0;12.1")
```

Comment:
Separator needs to be a `;` not a `,` in `...;10.3,11.0;...` for the compute capability autodetection to work.

Reply:
Yeah, good catch. Done!